com.unity.perception/CHANGELOG.md

```diff
@@ -9,12 +9,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Upgrade Notes
 
+All appearances of the term `KeyPoint` have been renamed to `Keypoint`. Therefore, the following steps will be required after upgrading to this version:
+* If you have code that relies on any renamed types or names, alter your code to reflect the new names.
+* If you are using custom `KeypointTemplate` assets besides the provided `CocoKeypointTemplate`, open those assets in a text editor and change the `keyPoints:` line to `keypoints:`, then save the file.
```

Comment:
Can you use `[FormerlySerializedAs("keyPoints")]` to get this automatically?

Reply:
yes! will do.